sympy/simplify/combsimp.py

```diff
@@ -217,6 +219,232 @@ def explicate(p):
                 denom_gammas.extend(l)
             elif isg is False:
                 denom_others.extend(l)
+        # =========== level >= 2 work: factor absorbtion =========
+        if level >= 2:
```

Comment:
Neither of these tests involve gammas. There should be new code dealing with pure combinatorial functions to implement these simplifications.